ztp/source-crs/HardwareEventRedfishSecret.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: redfish-basic-auth
+  annotations:
+    ran.openshift.io/ztp-deploy-wave: "100"
+type: Opaque
+data:
+  username: cm9vdA==
+  password: Y2Fsdmlu
```

Comment:
Maybe use sample/sample here, or use $user and $password. The $ syntax tells the generator to remove the line if not supplied by the user.

Reply:
Deleted the file. Since the data needs to be base64 encoded, I think it might be easier for customer just create the secret using literals from CLI. I will update documentation to let customer manually create secret like this:
```
oc -n openshift-hw-events create secret generic redfish-basic-auth --from-literal=username=$BMC_USERNAME --from-literal=password=$BMC_PASSWORD --from-literal=hostaddr="BMC_HOST_IP_ADDR"
```